runner.jl

```diff
@@ -0,0 +1,62 @@
+using Pkg
```

Comment:
I dont understand this file - do we need it? If I understand correctly the auto-format github action should resolve this - it will come soon :S (not in my hands, the maintainer needs to update something)

Reply:
no it is here accidentally
it is just my handle